solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/DesignSystem/DS+Font.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DS+Font.swift
+//  SpaceApp
+//
+//  Created by Pedro Gabriel on 14/03/22.
+//
+
+import SwiftUI
+
+extension DS {
+    public struct Font {
+        let font: SwiftUI.Font
```

Comment:
**[SUGGESTION]**
Don't call this font, prefer Typography. If you are using a font that is pre-defined in that system, it's ok to use font since the system itself already encapsulates the size and such, but if it's something new / customized it is not.
A font is just a small part of a typography definition, which can have: font, size, decorators (uppercased, strike through, etc)...